src/bin/defmt-print.rs

```diff
@@ -0,0 +1,55 @@
+use std::{
+    env, fs,
+    io::{self, Read},
+    path::PathBuf,
+};
+
+use anyhow::anyhow;
+use structopt::StructOpt;
+
+use probe_run::logger;
+
+/// Prints defmt-encoded logs to stdout
+#[derive(StructOpt)]
+struct Opts {
+    #[structopt(short, parse(from_os_str))]
+    elf: PathBuf,
```

Comment:
Is there a reason this is `-e` instead of a positional argument?

Reply:
Using a flag here would let us add an *optional* positional argument that specifies the defmt file to decode. That is these two would work:
``` console
$ cat /dev/ttyUSB0 | defmt-print -e my-elf
$ defmt-print -e my-elf some-defmt-data.bin
```

Reply:
> add an optional positional argument

add backwards compatibly in a patch release

Reply:
Ah, I see. I suppose we could also make the input file a named argument, but this works too.